### ISSUE TITLE
Send card to discard pile

### ISSUE THREAD

**Issue by mrcomac** [alessiocali/card-tiles#34]
When I send a card from my hand to the scene, i would like to that card be also send to the discard pile.

**Comment by kristianserrano** [comment]
Ideally, it should be added to a "table" pile as it's still in play and technically not yet discarded. If it's sent to the discard pile, one could possibly retrieve that card and play it to the table again. Even if there's scripting to remove the card from the scene when it's removed from the discard pile, that feels... incorrect?

**Comment by mrcomac** [comment]
i think i didn't understand your example, can you expand that please?

**Comment by kristianserrano** [comment]
There are games of various types that can allow one to pull discarded cards or to manipulate or take cards on a table. The discard pile is a very specific thing: these are cards that are no longer in play. They're not on the table, nor are they in anyone's hands.

Take Three-Dragon Ante, for example. There are ante cards that everyone puts down at the start of a gambit. These cards remain on the table until the gambit ends (three rounds of play). They can be taken by a player and added to their hand during play based on certain actions. At the end of the gambit, any remaining ante cards are then added to the discard pile.

Other games might have cards played to a table and allow draws from a discard pile. Cards on the table should not be included in draws from that discard pile because they are supposed to remain on the table.

**Comment by mrcomac** [comment]
I see. maybe it can be a configuration in the module, send cards to a pile or to a discard pile.

**Comment by kristianserrano** [comment]
That might be awkward if you're playing with a mixed use of cards in a game. I think a "table" pile is ideal, and when the cards on the table are finally discarded, they can be transferred from the table pile to the discard pile.

**Comment by mrcomac** [comment]
I disagree. Maybe for that use case makes sense but if I'm playing an adventure card to the table, and that is my "workflow" from hand to the table that means from the hand to discard, it will be an extra and not necessary complexity to create a table pile to work as a proxy to send cards to discard.
Since the system give us the power of configuration I can't see why not add that as a configuration :).

**Comment by kristianserrano** [comment]
The creation of the table pile and the moving the card from that pile to the discard pile can easily be automated. The user wouldn't have to actually do anything other than put the card on the table and remove it when done.
